CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [7.0.0]
+### Changed
+- **BREAKING:** Bump minimum Node version to 16 ([#139](https://github.com/MetaMask/json-rpc-engine/pull/139))
+- **BREAKING:** Use `@metamask/utils` ([#105](https://github.com/MetaMask/json-rpc-engine/pull/105))
+  - This package now uses `@metamask/utils` JSON-RPC types
+- Change package name to `@metamask/json-rpc-engine` ([#139](https://github.com/MetaMask/json-rpc-engine/pull/139))
+- Use `@metamask/rpc-errors` ([#138](https://github.com/MetaMask/json-rpc-engine/pull/138))
+
+### Added
```

Comment:
Looks like there was already an `Added` section

Reply:
I must have missed that 😅